### ISSUE TITLE
AndroidX support: Verso library overrides LazyCenteredViewPager

### ISSUE THREAD

**Issue by tasegula** [shopgun/shopgun-android-sdk#20]
ShopGun doesn't support AndroidX due to Verso dependency.

`Method 'void androidx.viewpager.widget.PagerAdapter.setViewPagerObserver(android.database.DataSetObserver)' is inaccessible to class 'androidx.core.view.CenteredViewPager'`

The problem is that in Verso you are overriding `LazyCenteredViewPager`, which confuses AndroidX.

Solution: change the package name of the `LazyCenteredViewPager` to not be the same as the one in Android Support.

Helpful links:
- The issue seems to be the same as here:
    - https://github.com/TangoAgency/material-intro-screen/issues/169
    - fixed in https://github.com/DreierF/material-intro-screen
    - by this commit https://github.com/DreierF/material-intro-screen/commit/663ece31c0cabd0809f74843a7c12a3d8301c62a#diff-dc95ead326172bef61210d4b34eefa3c

**Comment by tasegula** [comment]
The full stacktrace:
```
Process: app.demo, PID: 4542
    java.lang.IllegalAccessError: Method 'void androidx.viewpager.widget.PagerAdapter.setViewPagerObserver(android.database.DataSetObserver)' is inaccessible to class 'androidx.core.view.CenteredViewPager' (declaration of 'androidx.core.view.CenteredViewPager' appears in /data/app/app.demo-gbA9q2uIL1ooUgoUTPDI0Q==/split_lib_dependencies_apk.apk)
        at androidx.core.view.CenteredViewPager.setAdapter(CenteredViewPager.java:534)
        at androidx.core.view.LazyCenteredViewPager.setAdapter(LazyCenteredViewPager.java:29)
        at com.shopgun.android.verso.VersoViewPager.setAdapter(VersoViewPager.java:28)
        at com.shopgun.android.verso.VersoFragment.ensureAdapter(VersoFragment.java:481)
        at com.shopgun.android.verso.VersoFragment.notifyVersoConfigurationChanged(VersoFragment.java:460)
        at com.shopgun.android.sdk.pagedpublicationkit.PagedPublicationFragment.notifyVersoConfigurationChanged(PagedPublicationFragment.java:301)
        at com.shopgun.android.sdk.pagedpublicationkit.PagedPublicationFragment$PagedPublicationOnLoadComplete.onPublicationLoaded(PagedPublicationFragment.java:274)
        at com.shopgun.android.sdk.pagedpublicationkit.impl.apiv2.CatalogConfiguration$CatalogListener.ensurePublicationData(CatalogConfiguration.java:212)
        at com.shopgun.android.sdk.pagedpublicationkit.impl.apiv2.CatalogConfiguration$CatalogListener.onRequestIntermediate(CatalogConfiguration.java:199)
        at com.shopgun.android.sdk.pagedpublicationkit.impl.apiv2.CatalogConfiguration$CatalogListener.onRequestIntermediate(CatalogConfiguration.java:184)
        at com.shopgun.android.sdk.requests.LoaderDelivery$LoaderRequestPostBackRunnable.run(LoaderDelivery.java:78)
        at android.os.Handler.handleCallback(Handler.java:790)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Comment by AgneseBussone** [comment]
Solved in version `4.0.3` in f7ecf010ef5715042b469b5f1d6628cc9af79172
